index.html

```diff
@@ -89,48 +97,142 @@
 <script>
   var data = [];
   var options = {
-    yearRange: 'all'
+    party: 'all'
   };
-
-  var svg;
-
+  var svg, x, y;
+  var margin = { top: 15, right: 15, bottom: 30, left:45 };
+  var width = 640 - margin.right -  margin.left;
+  var height = 400 - margin.top - margin.bottom;
   // FETCH DATA
-
-  function setup() {
+  d3.json('data.json', function (error, json){
+    // Call back function
+    if (error) { throw error; }
+    data = json;
+    setup();
+    update();
+  });
+
+  function setup( ) {
     // SVG
-
     svg = d3.select('#chart')
-      .append('svg');
-
+      .append('svg')
+      .attr('width', width + margin.right + margin.left)
+      .attr('height', height + margin.top + margin.bottom)
+      .append('g')
+      .attr('transform','translate(' + margin.left + ',' + margin.top + ')');
+   
     // SCALES
-
+    x = d3.scale.linear()
+      .domain([d3.min(data,function (d){ return d.cfScore;}), Math.ceil(d3.max(data, function (d){ return d.cfScore; } )) ])
+        .range([margin.left,(margin.left + width)])
+        .nice();
+
+console.log(Math.ceil(d3.max(data, function (d){ return d.cfScore; } )))
+
+    y = d3.scale.linear()
+      .domain(d3.extent(data, function (d){ return d.distPart; } ))
+      .range([(height-margin.top),0])
+      .nice();    
+   
     // AXES
-
+    var xAxis = d3.svg.axis()
+      .scale(x)
+      .tickSize(-height)
+      .orient("bottom");
+    var yAxis = d3.svg.axis()
+      .scale(y)
+      .tickSize(-width)
+      .orient("left");
+    svg.append("g")
+      .classed("x axis", true)
+      .attr("transform", "translate(0," + (height-margin.top) + ")")
+      .call(xAxis);
+    svg.append("g")
+      .classed("y axis", true)
+      .attr("transform","translate(" + margin.left + ",0)")
+      .call(yAxis);
+    var line = d3.svg.line()
+      .x(function(d) { return xScale(d.x); })
+      .y(function(d) { return yScale(d.y); });
+    svg.append("text")
+        .attr("text-anchor", "middle")
+        .attr("transform", "translate(0,"+(height/2)+")rotate(-90)")
+        .text("District Partisanship");
+    svg.append("text")
+        .attr("text-anchor", "middle")
+        .attr("transform", "translate("+ (width/2) +","+(height + margin.top + (margin.bottom/2))+")")
```

Comment:
Looks like it's being clipped a bit. I see what you're doing with the math here. That makes sense, but remember that text is positioned from its top, not its baseline. You could leave this line unchanged and make an adjustment by using the `dy` attribute, which works great if you set it in ems. 1 em is equal to the height of a line of text, which is why you sometimes see things like this: `.attr('dy', '.71em')`.